components/director/internal/domain/systemauth/service.go

```diff
@@ -76,7 +82,7 @@ func (s *service) create(ctx context.Context, id string, objectType model.System
 
 	err = s.repo.Create(ctx, systemAuth)
 	if err != nil {
-		return "", errors.Wrapf(err, "while creating System Auth for %s", objectType)
+		return "", err
```

Comment:
Why remove the details of the error?

Reply:
I moved it in the [resolver](https://github.com/kyma-incubator/compass/blob/client-creds-pairing-logs/components/director/internal/domain/oauth20/resolver.go#L123) so that I can include both the object type and the object id in the error log. 

I have to remove the `model` part there as well based on your previous comment.